app/controllers/githook_controller.rb

```diff
@@ -43,4 +46,12 @@ def authenticate
   rescue ActionController::RedirectBackError
     redirect_to root_path
   end
+
+  def verify_signature(payload_body)
+    signature = 'sha1=' + OpenSSL::HMAC.hexdigest(
+      OpenSSL::Digest.new('sha1'),
+      ENV['SECRET_TOKEN'], payload_body)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.